### PR TITLE
linkedin

### PR DESCRIPTION
**Pull request by DevSoni2022**
As get to know new members' site soon deploy on Production,that issue already  fixed on new member site

### CODE REVIEW

File: src/components/social-media-icon/index.js [hunk 1]
```diff
@@ -6,6 +6,18 @@ import classNames from '@components/social-media-icon/social-media-icon.module.s
 const SocialMediaIcon = (props) => {
   const { id, type } = props;
 
+  let socialUserId = id;
+
+  if (
+    socialUserId.includes('linkedin') &&
```
Comment (shubhamjaintech) [inline src/components/social-media-icon/index.js:12]: if socialUserId is undefined/null for some reason, then all string fn i.e .includes can give error. 
Instead can we handle undefined/null case at  outser level only i.e line 10 with single if(socialUserId){} 
Thank you.

Comment (DevSoni2022) [reply src/components/social-media-icon/index.js]: Shubham same has resolved now.
Thanks

**Review by shubhamsigdar1 — COMMENTED** [review]
merge conflict
image

**Comment by DevSoni2022** [comment]
> merge conflict image

Merge conflict resolved

**Comment by shubhamjaintech** [comment]
@DevSoni2022 have we added any test case for this ticket/feature? if not then can we try adding it.? 
*I guess this change is similar of your old ticket related to twitter redirect, maybe if you added test case there, then can add similar test case for linkedin redirect. Thank you.

**Comment by DevSoni2022** [comment]
> @DevSoni2022 have we added any test case for this ticket/feature? if not then can we try adding it.? *I guess this change is similar to your old ticket related to Twitter redirect, maybe if you added a test case there, then can add a similar test case for LinkedIn redirect. Thank you.

Shubham, In my last task, also did not write any test case for specific card test cases already written.

**Comment by shubhamjaintech** [comment]
Approved via comment!

**Pull request closed without merging**